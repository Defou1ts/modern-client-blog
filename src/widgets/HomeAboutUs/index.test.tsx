import React from 'react';

import { render, screen } from '@testing-library/react';

import renderer from 'react-test-renderer';

import { HomeAboutUs } from './index';

describe('HomeAboutUs', () => {
	it('renders a  HomeAboutUs', () => {
		render(<HomeAboutUs />);
		const text = screen.getByText('home.about.about-title');
		expect(text).toBeInTheDocument();
		expect(text).toBeVisible();
	});

	it('HomeAboutUs snapshot', () => {
		const component = renderer.create(<HomeAboutUs />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
