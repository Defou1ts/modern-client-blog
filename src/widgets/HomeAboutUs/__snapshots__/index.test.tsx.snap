// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeAboutUs HomeAboutUs snapshot 1`] = `
<article
  className="wrapper"
  data-test-id="home-about-us"
>
  <div
    className="rectangles reverse rectangles"
  >
    <div
      className="purpleRectangle"
    />
    <div
      className="yellowRectangle"
    />
  </div>
  <div
    className="about"
  >
    <div
      className="item"
    >
      <p
        className="subtitle"
      >
        home.about.about-subtitle
      </p>
      <h2
        className="h h2 title"
      >
        home.about.about-title
      </h2>
      <p
        className="p medium description"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </p>
      <a
        className="link"
        data-test-id="home-about-us-link"
        href="/about"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        home.about.read-more
      </a>
    </div>
    <div
      className="item"
    >
      <p
        className="subtitle"
      >
        home.about.mission-subtitle
      </p>
      <h3
        className="h h3 title"
      >
        home.about.mission-title
      </h3>
      <p
        className="p medium description"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
      </p>
    </div>
  </div>
</article>
`;
